File: lingting-aws/src/main/java/live/lingting/framework/aws/policy/Acl.java
package live.lingting.framework.aws.policy;

/**
 * @author lingting 2024-09-12 20:55
 */
public enum Acl {

	PRIVATE("private"),

	PUBLIC_READ("public-read"),

	PUBLIC_READ_WRITE("public-read-write"),

	;

	private final String value;

	private Acl(String value) {
		this.value = value;
	}

	public String getValue() {return this.value;}
}

File: lingting-aws/src/main/kotlin/live/lingting/framework/aws/policy/Acl.kt
package live.lingting.framework.aws.policy

/**
 * @author lingting 2024-09-12 20:55
 */
enum class Acl(val value: String) {
    PRIVATE("private"),

    PUBLIC_READ("public-read"),

    PUBLIC_READ_WRITE("public-read-write"),
}
